src/plugins/timelist/inspector/TimelistPropertiesView.vue

```diff
@@ -30,6 +30,23 @@
             These settings don't affect the view while editing, but will be applied after editing is
             finished.
           </div>
+          <div class="c-inspect-properties__label" title="Display Style">Display Style</div>
+          <div v-if="canEdit" class="c-inspect-properties__value">
+            <select v-model="isExpanded" @change="updateExpandedView()">
+              <option
+                v-for="(displayStyleOption, index) in displayStyleOptions"
+                :key="index"
+                :value="displayStyleOption.key"
+              >
```

Comment:
since there are only 2 options, we should probably just hardcode this instead of using the v-for

Reply:
I'd prefer not to hard code this. Is there an issue with using the options like this?

Reply:
In my opinion it's much harder to read the code this way as the computed property, data property, and const array of objects all obfuscate the values. Hard-coding isn't always bad! :)

Another thought: this is a binary option, so from a web design perspective it should really be a checkbox, no?
@charlesh88 thoughts on that?

Reply:
It WAS a checkbox. Hehe. We changed it to a dropdown.

Reply:
I've just gone ahead and hard coded this in the interest of time.